modules/SofaGuiQt/src/sofa/gui/qt/SofaWindowProfiler.cpp

```diff
@@ -410,6 +410,21 @@ void SofaWindowProfiler::createTreeView()
     columnNames << "Hierarchy Step Name" << "Total (%)" << "Self (%)" << "Time (ms)" << "Self (ms)";
     tree_steps->setHeaderLabels(columnNames);
 
+    tree_steps->headerItem()->setToolTip(1, QString("Percentage of duration of this step compared to the duration of the root step"));
+    tree_steps->headerItem()->setToolTip(2,
+                                         QString("- If the step has child steps: percentage of the duration "
+                                                 "of this step minus the sum of durations of its children, compared to "
```

Comment:
Sure about this one?
I thought it was still the percentage of this step duration compare to parent step. Even if the step has children

Reply:
The code is here: https://github.com/sofa-framework/sofa/blob/291680b1950656586e4712eb0ab9f84818b1862a/modules/SofaGuiQt/src/sofa/gui/qt/SofaWindowProfiler.cpp#L116. What's your opinion?

Reply:
ah yes you are right. It is the % of this step without the children! gogogo